src/lib/calendar-nav/calendar-nav.component.less

```diff
@@ -53,7 +53,7 @@
 
     .dp-calendar-secondary-nav-right {
       left: initial;
-      right: 5px;
+      //right: 5px;
```

Comment:
I had to remove it to see the double arrow. When I first ran the code, arrows were one on the other, so we only saw one arrow. But it seems strange to be such a bug, so I put it in comments just in case. It'd be great if you could run the code and have a look to it (check/uncheck it in chrome to see the difference...)

Reply:
Will do